x-pack/plugins/alerting/server/routes/clone_rule.test.ts

```diff
@@ -97,7 +96,7 @@ describe('cloneRuleRoute', () => {
     },
     actions: [
       {
-        ...ruleToClone.actions[0],
+        ...omit(ruleToClone.actions[0], 'type'),
```

Comment:
`type` is not valid anymore. We should also add a test to verity that system actions are returned correctly.